extensions/spectacular/esbuild.js

```diff
@@ -3,120 +3,84 @@
 const { build } = require("esbuild");
 const glob = require("glob");
 const fs = require("fs-extra");
-const chokidar = require("chokidar");
-const path = require("path");
 
 const baseConfig = {
-  bundle: true,
-  minify: process.env.NODE_ENV === "production",
-  sourcemap: process.env.NODE_ENV !== "production",
-  loader: {
-    ".wasm": "file",
-  },
-  outbase: "src",
+	bundle: true,
+	minify: process.env.NODE_ENV === "production",
```

Comment:
Hello, these changes are not expected, as per `.editorconfig`:

https://github.com/meltylabs/melty/blob/9157a016b23359cf6feeb0d8b8b340f04ed66c6a/.editorconfig#L14-L15

Probably the editor is misconfigured and the indentation is mixing tabs and spaces

Reply:
This config is applied only to `[{*.yml,*.yaml,package.json}]`. In fact, in the same file, notice this block:
```
[*]
indent_style = tab
trim_trailing_whitespace = true
```

I guess my code editor is picking up this config for all files in general

Reply:
So the indentation is not being honored by existing code, maybe this should be addressed to reduced the number of changes in this PR?

Reply:
If I change this block in `.editorconfig`:
```
[*]
indent_style = tab
trim_trailing_whitespace = true
```
To: 
```
[*]
indent_style = space
indent_size = 2
trim_trailing_whitespace = true
```

Then it will create a lot more changes - this is because all the other files are formatted with the initial config. The indentation was incorrect in the `esbuild.js` file (formatted with spaces instead of tabs) and when my editor formatted the file, it used the correct editorconfig rules.

Reply:
Yeah, I think this is because we didn't reformat our code when we copied it into the vscode source repo. Would be good to go through and reformat it all at some point. For now we can live with a big diff.